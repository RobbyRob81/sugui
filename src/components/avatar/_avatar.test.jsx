import * as React from 'react';
import { customRender } from '../../../test/test-utils';
import Avatar from '.';

it('should render without throwing an error', () => {
  const { container } = customRender(<Avatar name="Error Found" />);
  expect(container.firstChild).toMatchSnapshot();
});
