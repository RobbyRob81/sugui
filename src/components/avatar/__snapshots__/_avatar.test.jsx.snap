// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without throwing an error 1`] = `
<div
  class="sc-bdVaJa dSMhla"
>
  <span
    class="sc-htpNat fbiTxL"
  >
    EF
  </span>
</div>
`;
